sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/csv/UnivocityParser.scala

```diff
@@ -82,7 +82,12 @@ class UnivocityParser(
 
   // Retrieve the raw record string.
   private def getCurrentInput: UTF8String = {
-    UTF8String.fromString(tokenizer.getContext.currentParsedContent().stripLineEnd)
+    val currentParsedContent = tokenizer.getContext.currentParsedContent()
+    if (currentParsedContent == null) {
```

Comment:
In which case you got `null` here?

Reply:
This was to get around a problem with CsvExpressionsSuite tests, which intentionally produces some bad records but also bypasses the tokenizer and calls UnivocityParser#convert directly. This was not a problem before, because those tests did not specify a corrupt records field.